homeassistant/components/ambient_station/__init__.py

```diff
@@ -42,6 +44,20 @@
 CONFIG_SCHEMA = cv.deprecated(DOMAIN)
 
 
+def async_lx_to_perceived(value: int) -> int:
+    """Calculate illuminance (in lux)."""
+    lux = async_wm2_to_lx(value)
+
+    try:
+        perceived = round(math.log10(lux) / 5) * 100
```

Comment:
I think this is borderline calculating a new state which device and service integrating integrations should not do. The standard is to set the state that is reported by the device or service while following our entity design.

Reply:
Understood; I'll close this.